sdk/trts/trts_veh.cpp

```diff
@@ -251,6 +259,29 @@ static void apply_constant_time_sgxstep_mitigation_and_continue_execution(sgx_ex
         }
     }
 
+    ct_result = ct_decode(&info->cpu_context, &data_address);
```

Comment:
ct_decode is missing in simulation build

Reply:
I wasn't able to replicate this issue, but I suspect that the root cause is that ctd.c was in `sdk/trts/linux`, and the Makefile in that directory configures the linking a little differently from the rest of the trts.

I updated the PR to move ctd.c up into `sdk/trts`. Please try the build again and let me know if it still fails.

Reply:
when you build Sample Enclave, use make SGX_MODE=SIM, it will report ct_decode is undefined.

Reply:
It looks better if we can remove all the AEX-Notify related binaries from simulation.  Right now, we just disable it during runtime while the AEX-Notify logic is still included in the simulation library. 
@scottconstable you can temporarily update https://github.com/intel/linux-sgx/blob/master/sdk/simulation/trtssim/linux/Makefile#L61 to add the new ctd.o in the simulation library to let the simulation mode compilation pass.  We will consider to remove all the AEX-Notify logic from simulation after your PR.

Reply:
Thanks! I took Lili's advice and simply updated the trtssim Makefile, and I no longer see the build failing on `ct_decode` with `SGX_MODE=SIM`.